contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_present_in_other_table.py

```diff
@@ -163,7 +163,8 @@ def _validate(
         self._validate_template_dict(configuration)
         final_value = metrics.get("query.template_values")[0]["COUNT(1)"]
         return ExpectationValidationResult(
-            success=(final_value == 0), result={"observed_value": final_value}
+            success=(final_value == 0),
+            result={"observed_value": f"{final_value} missing items"},
```

Comment:
```suggestion
            result={"observed_value": f"{final_value} missing item{'s' if final_value != 1 else ''}"}
```

(this way if there is 1 item then it'll say "1 missing item" instead of "1 missing items")

Reply:
Brilliant! thank you @JennyTee